coffeecard/CoffeeCard/Models/Product.cs

```diff
@@ -6,13 +6,20 @@ public class Product
     {
         public int Id { get; set; }
 
-        [Required] public int Price { get; set; }
+        [Required]
+        public int Price { get; set; }
 
-        [Required] public int NumberOfTickets { get; set; }
+        [Required]
+        public int NumberOfTickets { get; set; }
 
-        [Required] public string Name { get; set; }
+        [Required]
+        public string Name { get; set; }
 
         public string Description { get; set; }
         public int ExperienceWorth { get; set; }
+        [Required]
+        public bool Visible { get; set; }
+        [Required]
+        public bool BaristasOnly { get; set; }
```

Comment:
We may want this as an enum, considering that different people has different prices, i.e board vs baristas and a few special people